### PR TITLE
Scala 2.13.0-RC1

### PR DESCRIPTION
**Pull request by xuwei-k**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -39,10 +39,17 @@ lazy val baseSettings = Seq(
...
   transitiveClassifiers in Global := Seq(Artifact.SourceClassifier),
-  scalatestVersion := "3.0.7",
+  scalatestVersion := "3.0.8-RC2",
   specs2Version := "4.5.1",
-  parserCombinatorsVersion := "1.1.1",
-  collectionCompatVersion := "0.3.0",
+  parserCombinatorsVersion := {
+    CrossVersion.partialVersion(scalaVersion.value) match {
+      case Some((2, 11)) =>
+        "1.1.1"
```
Comment (xuwei-k) [inline build.sbt]: https://github.com/scala/scala-parser-combinators/issues/200

**Comment by seratch** [comment]
TODO: 3.3.x backport

**Comment by seratch** [comment]
As #1002 shows, it seems to be impossible to release a new 3.3x version with Scala 2.13.0-RC1. I will work on 3.4.0 release soon.

**Comment by xuwei-k** [comment]
Maybe we can keep binary compatibility if old version `scala-collection-compat` with Scala 2.11, 2.12 🤔

**Comment by seratch** [comment]
Ah, that's true. Will try it out later.

**Pull request merged**